dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -349,11 +353,16 @@ private Collection getAndCheckCollection(Optional<Long> organizationId, Long col
         }
 
         List<Collection> collections = collectionDAO.findAllByOrg(organizationId);
-
+        Session currentSession = sessionFactory.getCurrentSession();
         if (checkIncludes(include, "entries")) {
             collections.forEach(collection -> Hibernate.initialize(collection.getEntries()));
```

Comment:
Seems like the entries were coming back anyway without this initialize, so is this even necessary?

Reply:
Right now, it's not. but if we fix the lazy loading, it's going to be. there's not much reason to remove it right now.